bundle/manifests/logging.openshift.io_kibanas_crd.yaml

```diff
@@ -156,7 +156,55 @@ spec:
           - managementState
           type: object
         status:
-          type: object
+          items:
+            description: KibanaStatus defines the observed state of Kibana
+            properties:
+              clusterCondition:
+                additionalProperties:
+                  items:
+                    properties:
+                      lastTransitionTime:
+                        description: Last time the condition transitioned from one
+                          status to another.
+                        format: date-time
+                        type: string
+                      message:
+                        description: Human-readable message indicating details about
+                          last transition.
+                        type: string
+                      reason:
+                        description: Unique, one-word, CamelCase reason for the condition's
```

Comment:
nit: ~CamelCase~ PascalCase

Reply:
No chance to change this wording. It comes from deps.